gradle.properties

```diff
@@ -1,7 +1,7 @@
 # artifacts group
 group=ai.h2o
 # Major version of H2O release
-h2oMajorVersion=3.32.0
+h2oMajorVersion=3.33.0
```

Comment:
note: Don't forget to revert this line before merge

Reply:
I made this Draft not to be mergable by accident.